### PR TITLE
feat(elements): Sign in resendable verifications [SDK-1582]

### PR DESCRIPTION
**Pull request by tmilewski**
## Description

Adds resendable verifications to sign-in.

SDK-1582

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [x] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

File: packages/elements/src/react/sign-in/action.tsx [hunk 1]
```diff
@@ -1,21 +1,104 @@
...
+  },
 );
 
+SignInResend.displayName = SIGN_IN_RESEND_NAME;
+
+const SIGN_IN_ACTION_NAME = 'SignInAction';
+
+export function Test() {
+  return <SignInResend fallback={({ resendableAfter }) => <p>Resendable in: {resendableAfter}s</p>}>Foo</SignInResend>;
+}
```
[the remark this answers is not in the document]
Comment (tmilewski) [reply packages/elements/src/react/sign-in/action.tsx:100]: Haha, whoops, yes.

**Review by BRKalow — APPROVED** [review]
lgtm 👍

**Review by alexcarpenter — COMMENTED** [review]
Do you also get this type error for using the `fallback` prop?

Screenshot 2024-04-11 at 9 24 19 AM

**Comment by alexcarpenter** [comment]
> > Do you also get this type error for using the `fallback` prop?
> 
> @alexcarpenter Nope, I didn't. I'll check again.

Must just be a me thing then 👍🏼

**Review by alexcarpenter — APPROVED** [review]

**Pull request merged**